### DISCUSSION TITLE
Welcome to OSS Project Ideas Discussion!

### DISCUSSION THREAD

**Discussion by sloorush** [Dyalog/GSOC#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Share ideas.
  * All projects will have their own discussions and a new discussion would be open for new project ideas that the community wants to suggest.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Great Insights

**Comment by adityadeshlahre** [comment]
Hello There,
I am Aditya, a CS student from India!
I like contributing to open-source repositories and learning from the community.

currently seems like "Dyalog" is focusing on maintaining proficiency fo Language and making it more accessible to Open Source community!

Tbh, I don't have ideas to discuss but creating APL with the Support of IDE is a big achievement.

In a while, I had a Question "Dyalog" doesn't have its own deployed/live/usable version in public ??

Thank You

**Reply by ghost** [reply]
Hey! Aditya
Dyalog APL is a commercial product, but they do offer a free Personal/Non-Commercial Edition for students and hobbyists to learn and experiment with. If you’re asking about something live or usable without installation, they’ve got TryAPL, which is an online tool where you can write and test APL code directly in your browser.

Hope this clears it up! Let me know if you have more questions.

Cheers,
Aditya

**Reply by sloorush** [reply]
Hi @adityadeshlahre 
If you want to try Dyalog APL on its own you can download it from here: https://www.dyalog.com/download-zone.htm

To learn and you can also checkout: https://xpqz.github.io/learnapl/intro.html

You can also talk to the community and ask questions on: https://apl.chat

Thanks! Let us know if you would like to know more.

**Comment by Shashank-248** [comment]
Hey!
I was looking out for GSOC projects and came across Dyalog. The ideas aren't finalized and announced yet. Is Dyalog not participating in GSOC-2025?
Thanks.

**Reply by sloorush** [reply]
Hi @Shashank-248 
Thank you for your question. According to the [GSoC 2025 timeline](https://developers.google.com/open-source/gsoc/timeline). The applications for organisations started on 27th of January and end on 11th of February. We will be applying for GSoC. The final list of ideas will be published before February 11 - 18:00 UTC. GSoC will make the decision to accept us by February 27 - 18:00 UTC when the list of accepted mentoring organisations is published.

Should we not get accepted, these idea will still be available to work on by anyone in the community who wants to work on them.

In conclusion, stay tuned, we will know if we get accepted by the end of February 2025. 🚀
